single_group_playbooks/irods.yml

```diff
@@ -1,29 +1,22 @@
 ---
 - import_playbook: pre_deploy_checks.yml
 
-- name: 'Sanity checks before we start.'
-  hosts: irods
-  pre_tasks:
-    - name: 'Verify Ansible version meets requirements.'
-      assert:
-        that: "ansible_version.full is version_compare('2.10', '>=')"
-        msg: 'You must update Ansible to at least 2.10.x to use this playbook.'
-- name: 'Satisfy dependencies for and install Pulp.'
+- name: 'Satisfy dependencies for and install iCAT'
   hosts: irods
   roles:
-    - admin_users
-    - ssh_host_signer
-    - ssh_known_hosts
-    - pulp_client
-    - static_hostname_lookup
-    - logrotate
-    - logins
-    - sshd
-    - {role: geerlingguy.security, become: true}
-    - {role: geerlingguy.firewall, become: true}
-    - remove
-    - update
-    - docker
+#    - admin_users
+#    - ssh_host_signer
+#    - ssh_known_hosts
+#    - pulp_client
+#    - static_hostname_lookup
+#    - logrotate
+#    - logins
+#    - sshd
+#    - {role: geerlingguy.security, become: true}
+##    - {role: geerlingguy.firewall, become: true}
+#    - remove
+#    - update
+#    - docker
     - irods
-    - irods_davrods
+#    - irods_davrods
```

Comment:
Accidentally committed comments used for debugging only the `irods_davrods` role?